i18n/keys.pot

```diff
@@ -17,308 +17,301 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 
-#: Internet/Internet Explorer 6.0/Online/script.js:69
```

Comment:
That's really strange. Like the script walks your directory in a different way than on my machine.